shuffle-coordinator/src/main/java/com/alibaba/flink/shuffle/coordinator/highavailability/zookeeper/ZooKeeperMultiLeaderRetrievalDriver.java

```diff
@@ -83,35 +85,43 @@ public ZooKeeperMultiLeaderRetrievalDriver(
         client.getUnhandledErrorListenable().addListener(this);
         this.pathChildrenCache = new PathChildrenCache(client, "/", true);
         pathChildrenCache.getListenable().addListener(this);
-        pathChildrenCache.start(PathChildrenCache.StartMode.BUILD_INITIAL_CACHE);
-        mayUpdateLeader(pathChildrenCache.getCurrentData());
+        pathChildrenCache.start(PathChildrenCache.StartMode.POST_INITIALIZED_EVENT);
```

Comment:
Here I don't get why do we modify the `StartMode`?  Because the data cache size is not that much, I think `StartMode.BUILD_INITIAL_CACHE` can also cache the data quickly enough. Do you think so?